src/services/phantomApi.js

```diff
@@ -0,0 +1,25 @@
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react';
+
+const currentLang = localStorage.getItem('language')
+const phantomApiHeaders = {
+  'content-type': 'application/json',
+  'Connection': 'keep-alive',
+  'Accept-Language': currentLang
+}
+
+const baseUrl = 'https://phantom-api-pirates.herokuapp.com/';
```

Comment:
Make this an environment variable so that it can't be seen directly in the codebase